### ISSUE TITLE
Do not show "add search engine" if susper is already added as search engine

### ISSUE THREAD

**Issue by mariobehling** [fossasia/susper.com#196]
If Susper is already the search engine the dialog to install susper is still appearing on the front page. Expected: Only show "add search engine" if Susper is not yet installed in browser.

**Comment by harshit98** [comment]
@nikhilrayaprolu  @Marauderer97 `IsSearchProviderInstalled` will return always 0. Read more about it here - https://www.chromestatus.com/feature/5672001305837568 . You can also try `external.IsSearchProviderInstalled("https://www.google.com");` in the console. It will return always undefined value. What we can do is remove the default box completely as already we have added the auto-discovery feature in PR #362

**Comment by nikhilrayaprolu** [comment]
@harshit98 if we remove the box?How can we encourage a user in making susper as his default box?
can you suggest some alternative for encouraging users to add susper as their default search engine?

**Comment by harshit98** [comment]
@nikhilrayaprolu you're right. I'll see if I'll get any other alternative.

**Comment by harshit98** [comment]
@nikhilrayaprolu @Marauderer97 We can achieve this by storing cookies.

**Comment by nikhilrayaprolu** [comment]
@harshit98 that is not a permanent solution, what if the user clears cookies.

**Comment by harshit98** [comment]
@nikhilrayaprolu Then again it should pop up. Google behaves similarly like this. :)

**Comment by nikhilrayaprolu** [comment]
@harshit98 I don't find google behaving such if the user has already installed the search extension, why should we show the pop up after clearing cookies?

**Comment by vaibhavsingh97** [comment]
Claiming this issue

**Comment by harshit98** [comment]
Sure @vaibhavsingh97. Please go ahead. Kindly while making a PR for it, follow up the best practices.

**Comment by AakashMallik** [comment]
Since this issue has not been touched for roughly an year now... I am picking it up from here. @vaibhavsingh97  @harshit98  @nikhilrayaprolu

**Comment by praveenojha33** [comment]
@AakashMallik Sure go ahead

**Comment by sk9331657** [comment]
@AakashMallik  Are u working on it?? Can i take up this issue??

**Comment by AakashMallik** [comment]
@sk9331657 Yeah sure, I have my exams so I won't be able to work on it as of now.
